single_kernel_mongo/templates/enable-transparent-huge-pages.service.j2

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Enable Transparent Hugepages (THP)
+DefaultDependencies=no
+After=sysinit.target local-fs.target
+Before={{service_file}}
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c 'echo always | tee /sys/kernel/mm/transparent_hugepage/enabled > /dev/null && echo defer+madvise | tee /sys/kernel/mm/transparent_hugepage/defrag > /dev/null && echo 0 | tee /sys/kernel/mm/transparent_hugepage/khugepaged/max_ptes_none > /dev/null && echo 1 | tee /proc/sys/vm/overcommit_memory > /dev/null'
```

Comment:
2 things here:
1. Can you apply `/proc/sys/vm/overcommit_memory` through the sysctl command setting? (and handle failure if any in the likes of LXD)
2. Can you break down the the `ExecStart` into multiple `ExecStartPre` commands for readability?
3. Can you add the `-` character after the equal to tolerate failures in case they don't apply, think LXD ?

Reply:
Ok